src/main/java/de/tum/in/www1/artemis/repository/StatisticsRepository.java

```diff
@@ -97,6 +169,15 @@
             """)
     List<Map<String, Object>> getActiveTutors(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate);
 
+    @Query("""
+            select r.completionDate as day, r.assessor.login as username
+            from Result r
+            where (r.assessmentType = 'MANUAL' or r.assessmentType = 'SEMI_AUTOMATIC') and r.completionDate >= :#{#startDate} and r.completionDate <= :#{#endDate} and r.assessor.login not like '%test%'
+            and r.participation.exercise.id in :exerciseIds
+            """)
+    List<Map<String, Object>> getActiveTutorsForCourse(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate,
```

Comment:
please use a custom object here and not a map

Reply:
Done